biojava-structure/src/main/java/org/biojava/nbio/structure/io/PDBFileParser.java

```diff
@@ -1825,30 +1826,48 @@ private void  pdb_ATOM_Handler(String line)	{
 
 		// Parse element from the element field. If this field is
 		// missing (i.e. misformatted PDB file), then parse the
-		// name from the atom name.
+		// element from the chemical component.
 		Element element = Element.R;
+		boolean guessElement = true;
 		if ( line.length() > 77 ) {
 			// parse element from element field
-			try {
-				element = Element.valueOfIgnoreCase(line.substring (76, 78).trim());
-			}  catch (IllegalArgumentException e){}
+			String elementSymbol = line.substring(76, 78).trim();
+			if (elementSymbol.equals("")) {
+				logger.warn("Element column was empty. Assigning atom element "
+						+ "from Chemical Component Dictionary information", elementSymbol);

```

Comment:
The `elementSymbol` parameter is superfluous here. In any case `elementSymbol` is empty so there's not point in printing it out. It would be nice if some more info like atom name and serial could be output in the warning.